README.md

```diff
@@ -19,6 +22,27 @@
 
 <a href='https://testflight.apple.com/join/WiN0G62Q'><img height='70' alt='Join the Beta on TestFlight' src='https://anotherlens.app/testflight-badge.png'/></a>
 
+**Don't see Swiftfin in your language?**
```

Comment:
SwiftFin, capital F

Reply:
That's all I can see. Looks good otherwise :)

Reply:
fixed XD